src/excmds.ts

```diff
@@ -4360,6 +4360,48 @@ export async function echo(...str: string[]) {
     return str.join(" ")
 }
 
+/** helper function for js and jsb
+ *
+ * -p to take an extra argument located at the end of str[]
+ * -s to load js script of a source file from the config path
```

Comment:
Thanks for this PR. I don't think I like this "relative path-ness" by default. Could you perhaps add a `-r` option for paths relative to rc and have `-s` use the "normal" path behaviour (which is relative to the native messenger's installation directory). I think relative to RC by default could be a little surprising.

Why do you say that "/../" can't be in the path? It should "just work" with Native.read and resolve it to wherever that path is. If it doesn't, that's a bug :)

Reply:
The /../ was a safety precaution to prevent the script descent to arbitrary file in users system, like /etc/passwd (well that particular file was less important now a day.)

If you think that Native already does arbitrary things on user system, it doesn't matter. The check can be removed.

Reply:
I think I want people to be able to execute arbitrary files on their system, which Native.read already allows :)

Tridactyl's security model is this:

- the command line is isolated from web pages in an iframe and so Tridactyl's commands are not accessible to the web page
- key events are either real key events from users or fake key events triggered by Tridactyl
- users are trusted

So I don't see an issue with having it. Thanks for caring about security!